app/models/Ticket.js

```diff
@@ -1,23 +1,40 @@
-import mongoose, { Schema } from "mongoose";
-
-mongoose.connect(process.env.MONGODB_URI);
-mongoose.Promise = global.Promise;
-
-const ticketSchema = new Schema(
-  {
-    title: String,
-    description: String,
-    category: String,
-    priority: Number,
-    progress: Number,
-    status: String,
-    active: Boolean,
-  },
-  {
-    timestamps: true,
+import mongoose, { mongo, Schema } from "mongoose";
+let Ticket;
```

Comment:
Can you explain what benefit would this change provide over using `const Ticket`.
I am under the impression that we never want to update the Ticket object, do you believe otherwise?

Reply:
I just did it so that in Rout.js I can send a specific response from server when server was unable to  featch tickets ( by raising an exception that tickets are null further handling it and sending a ticket not found response to frontend)